Automattic/wp-calypso
# Color Schemes: Add Powder Snow color scheme

#### Changes proposed in this Pull Request

* Adds a new color scheme to the admin, "Powder Snow".

<img width="1278" alt="screen shot 2019-02-01 at 9 55 00 am" src="https://user-images.githubusercontent.com/2124984/52130190-833dc200-2607-11e9-9843-665291e43927.png">

#### Testing instructions

* Switch to this PR
* Navigate to `/me/account` and switch color scheme to "Powder Snow"
* Check throughout Calypso for contrast issues and other visual bugs
